astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -1507,6 +1514,7 @@ def __init__(
         col_offset=None,
         parent=None,
         *,
+        doc_node: Optional[Const] = None,
```

Comment:
Can you move it after `end_col_offset`? Both `end_lineno` and `end_col_offset` are direct ast attributes so I think it makes sense to keep them first.

Please move the docstring as well.